app/src/main/res/values-sk/strings.xml

```diff
@@ -0,0 +1,349 @@
+    <string name="tx_destination">Cieľ</string>
+    <string name="tx_destination_btc">Cieľová (BTC)</string>
+    <string name="tx_paymentId">ID Platby</string>
+    <string name="tx_blockheight">Blok</string>
+    <string name="tx_amount">Suma</string>
+    <string name="tx_amount_btc">Čiastka\n(BTC)</string>
+    <string name="tx_fee">Poplatok</string>
+    <string name="tx_transfers">Transfery</string>
+    <string name="tx_notes">Poznámky</string>
+    <string name="tx_notes_hint">(voliteľné)</string>
+    <string name="tx_button_notes">Ulož</string>
+    <string name="tx_notes_set_failed">Uloženie poznámky zlyhalo</string>
+    <string name="tx_title">Detaily Transakcie</string>
+
+    <string name="tx_pending">ČAKAJÚCI</string>
+    <string name="tx_failed">ZLYHAL</string>
+
+    <string name="receive_paymentid_hint">ID Platby (voliteľné)</string>
+    <string name="receive_amount_hint">Suma</string>
+    <string name="receive_cannot_open">Nemohol som otvoriť peňaženku!</string>
+    <string name="receive_paymentid_invalid">16 alebo 64 hexa znakov (0&#8211;9,a&#8211;f)</string>
+    <string name="receive_integrated_paymentid_invalid">Musí byť prázdna s integrovanou adresou</string>
+
+    <string name="receive_amount_too_big">Max. %1$s</string>
+    <string name="receive_amount_negative">Min. 0</string>
+    <string name="receive_amount_nan">XMR nie číslo</string>
+
+    <string name="receive_title">Prijať</string>
+
+    <string name="details_alert_message">Budú zobrazené citlivé dáta.\nPozri cez plece!</string>
+    <string name="details_alert_yes">Som v bezpečí</string>
+    <string name="details_alert_no">Poď späť!</string>
+    <string name="details_title">Detaily</string>
+
+    <string name="archive_alert_message">Peňaženka bude zálohovaná a následne zmazaná!</string>
+    <string name="archive_alert_yes">Áno, poďme na to!</string>
+    <string name="archive_alert_no">Nie, díky!</string>
+
+    <string-array name="priority">
+        <item>Priorita Štandard</item>
+        <item>Priorita Nízka</item>
+        <item>Priorita Medium</item>
+        <item>Priorita Vysoká</item>
+    </string-array>
+
+    <string name="fab_create_new">Vytvor novú peňaženku</string>
+    <string name="fab_restore_viewonly">Obnov prezeraciu peňaženku</string>
+    <string name="fab_restore_key">Obnov peňaženku zo súkromných kľúčov</string>
+    <string name="fab_restore_seed">Obnov peňaženku z 25-slovného seedu</string>
+
+    <string name="accounts_drawer_new">Vytvor Účet</string>
+    <string name="accounts_drawer_title">Účty</string>
+    <string name="accounts_new">Pridaný nový účet #%1$d</string>
+    <string name="tx_account">Účet #</string>
+
+    <string name="send_sweepall">Pošli všetky potvrdené prostriedky na tomto účte!</string>
+    <string name="tx_subaddress">Podadresa #%1$d</string>
+    <string name="generate_address_label_sub">Verejná Podadresa #%1$d</string>
+	
+	<string name="language">Slovenský</string>
```

Comment:
this was removed in master - please remove it here.

Reply:
Thank you @m2049r  for review. 
I assume, that you mean just one string 
 `<string name="language"> `
It has been removed already. Sorry for any inconveniences caused.